azure-mgmt-network/src/main/java/com/microsoft/azure/management/network/NetworkInterface.java

```diff
@@ -0,0 +1,389 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.management.network;
+
+import com.microsoft.azure.CloudException;
+import com.microsoft.azure.management.network.implementation.api.NetworkInterfaceInner;
+import com.microsoft.azure.management.resources.fluentcore.arm.models.GroupableResource;
+import com.microsoft.azure.management.resources.fluentcore.arm.models.Resource;
+import com.microsoft.azure.management.resources.fluentcore.model.Refreshable;
+import com.microsoft.azure.management.resources.fluentcore.model.Wrapper;
+import com.microsoft.azure.management.resources.fluentcore.model.Updatable;
+import com.microsoft.azure.management.resources.fluentcore.model.Creatable;
+import com.microsoft.azure.management.resources.fluentcore.model.Appliable;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Network interface.
+ */
+public interface NetworkInterface extends
+        GroupableResource,
+        Refreshable<NetworkInterface>,
+        Wrapper<NetworkInterfaceInner>,
+        Updatable<NetworkInterface.Update> {
+
+    /**
+     * @return <tt>true</tt> if IP forwarding is enabled in this network interface
+     */
+    boolean isIPForwardingEnabled();
```

Comment:
we should stick to proper camel casing, regardless of what the auto-gen'd Api uses. So this would be:
isIpForwardingEnabled().
Although this convention isn't super well documented in Java, there is a good discussion of it on stockoverflow - mostly borrowing from .NET's naming convention, which is more granular:
http://stackoverflow.com/questions/2236807/java-naming-convention-with-acronyms